src/c++/uds/src/uds/murmurhash3.c

```diff
@@ -8,7 +8,24 @@
 
 #include "murmurhash3.h"
 
+#ifndef VDO_UPSTREAM
+#include <linux/version.h>
+#undef VDO_USE_NEXT
+#if defined(RHEL_RELEASE_CODE) && defined(RHEL_MINOR) && (RHEL_MINOR < 50)
+#if (RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(10, 0))
```

Comment:
I suspect the RHEL_RELEASE_CODE handling is also not correct, but we can address that later when (as I expect) this code gets pulled into RAWHIDE. (But it should probably be >= 10.0 since this will get pulled into the RHEL 10 streams once it's released.)